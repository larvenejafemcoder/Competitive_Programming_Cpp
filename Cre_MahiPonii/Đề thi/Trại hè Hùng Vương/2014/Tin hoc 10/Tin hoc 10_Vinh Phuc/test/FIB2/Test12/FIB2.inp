23
21 6272
31 183613
38 26849402
26 119060
35 11285228
35 3227440
17 1215
22 12014
28 182038
13 132
0 1
1 1
35 6360295
30 944611
32 2301856
18 814
23 37806
24 62669
6 8
39 22847053
19 4744
30 1014008
15 677
