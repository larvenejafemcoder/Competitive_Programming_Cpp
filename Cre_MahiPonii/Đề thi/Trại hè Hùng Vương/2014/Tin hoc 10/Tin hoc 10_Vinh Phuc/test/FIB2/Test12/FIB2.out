2396
70134
10255559
45477
4310574
1232773
464
4589
69532
51
1
0
2429417
360809
879231
311
14441
23938
3
8726798
1812
387317
259
